pyproject.toml

```diff
@@ -42,6 +42,7 @@ openai = ["openai >= 1.0.0"]
 vllm = ["vllm >= 0.2.1"]
 vertexai = ["google-cloud-aiplatform >= 1.38.0"]
 together = ["together"]
+anthropic =["anthropic >= 0.20.0"]
```

Comment:
This should be ordered alphabetically i.e. `anthropic` and `argilla` should go on top of the list of extras (I believe the rest are more or less sorted, but feel free to do so for the rest too)